_data/simple-icons.json

```diff
@@ -5005,6 +5005,11 @@
             "hex": "CB3837",
             "source": "https://github.com/npm/logos"
         },
+        {
+            "title": "Nrwl",
+            "hex": "96D7E8",
+            "source": "https://nrwl.io/assets/nrwl-logo-white.svg"
```

Comment:
Could you please update the source to the homepage? The SVG itself is embedded in the header, and will likely be kept that way even if the file path changes in the future.

Reply:
Sure will do!